bundle/tests/scorecard/kuttl/test-multi-app-interface-db/01-pods.yaml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdEnvironment
+metadata:
+  name: test-multi-app-interface-db
+spec:
+  targetNamespace: test-multi-app-interface-db
+  providers:
+    web:
+      port: 8000
+      mode: operator
+    metrics:
+      port: 9000
+      mode: operator
+      path: "/metrics"
+    kafka:
+      namespace: kafka
+      clusterName: my-cluster
+      mode: none
+    db:
+      imageList: 
+        - version: 12
+          image: "registry.redhat.io/rhel8/postgresql-12:1-36"
```

Comment:
Do we want to swap these to the quay/postgresql-rds images?

Reply:
yeh probably makes sense
Can you give me the image name and I'll replace it

Reply:
`quay.io/cloudservices/postgresql-rds:12-1`
`quay.io/cloudservices/postgresql-rds:10-1`